conf/application.conf

```diff
@@ -171,3 +171,11 @@ gov-uk.guidance {
   agentServices.url = "https://www.gov.uk/guidance/get-an-hmrc-agent-services-account"
   mtdVat.url = "https://www.gov.uk/guidance/making-tax-digital-for-vat"
 }
+
+features {
+  staticDate.enabled = true
```

Comment:
Are there corresponding app-config PR's for this

Reply:
Its in the description - hmrc/app-config-production#3871 :)